src/components/addon/addon.jsx

```diff
@@ -5,21 +5,29 @@ import classNames from 'classnames';
 /**
   This component is not intended for public use
 */
-export default function Addon({ className, children, ...rest }) {
-  const classes = classNames('addon', className);
+export default function Addon({ addon, className, position, separator, ...rest }) {
+  if (addon === null) {
```

Comment:
Since addon is type "node", I would prefer addon to be undefined as default, which mean we don't need defaultProps and then we can have `if (!addon) {`, which will catch both null and undefined.

Reply:
That is more like it did before, but I opted to change that to allow falsy values as well, such as `0`. But maybe I should change it to `if (typeof addon !== 'undefined') {` (but then I wont catch `null`, which should be ok since null will not be rendered anyway :)

Reply:
addon have the propType node so it shouldn't get have gotten 0 or false

Reply:
I think you may be referring to the `element` prop-type? Because plain strings and/or numbers are valid `node`s according to [React docs](https://reactjs.org/docs/typechecking-with-proptypes.html): 

> ...
> // Anything that can be rendered: numbers, strings, elements or an array
> // (or fragment) containing these types.
> optionalNode: PropTypes.node,

Reply:
aha yeah, true, was thinking of element, sorry